selfdrive/controls/lib/latcontrol_pid.py

```diff
@@ -42,6 +42,7 @@ def update(self, active, CS, CP, VM, params, desired_curvature, desired_curvatur
       output_steer = self.pid.update(angle_steers_des, CS.steeringAngleDeg, check_saturation=check_saturation, override=CS.steeringPressed,
                                      feedforward=steer_feedforward, speed=CS.vEgo, deadzone=deadzone)
       pid_log.active = True
+      pid_log.angleError = angle_steers_des - CS.steeringAngleDeg
```

Comment:
Think we can always set the value, even when the controller is not active. Just move this line to above the if statement.